addons/calendar/models/mail_activity.py

```diff
@@ -0,0 +1,41 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import api, models, fields, tools, _
+
+
+class MailActivityType(models.Model):
+
+    _inherit = "mail.activity.type"
+
+    meeting_type = fields.Boolean(string='Manage through Calendar')
+
+class MailActivity(models.Model):
+
+    _inherit = "mail.activity"
+
+    activity_meeting_type = fields.Boolean(related='activity_type_id.meeting_type')
+    calendar_event_id = fields.Many2one('calendar.event', string="Calendar Meeting", compute='_compute_calendar_event_id')
```

Comment:
On second thought, this should be a stored Many2one field and the `activity_id` field from `calendar.event` should be a One2many (and renamed `activity_ids`).

The One2many on events means it's not an extra column on the table, and since there will be a lot more events than activities, that's a good thing :-) (activities are deleted upon completion, events aren't)